src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/adapters/AbstractTableAdapter.js

```diff
@@ -40,6 +41,15 @@ export default class AbstractTableAdapter extends AbstractAdapter {
 
             return (
                 <Table.Cell key={item.id + schemaKey}>
+                    {index === 0 && !item.ghostLocale
```

Comment:
I feel like we should make sure, that we are always referencing the `index` of 0, shouldn't we? Would imagine that somehow like this:

```javascript
<Table.Cell>
    {index === 0 &&
        item.ghostLocale
            ? <GhostIndicator ... />
            : (draft || ! published)
                ? <PublishIndicator />
                : null
    }
</Table.Cell>
```

(Didn't implement it all way through)